common/config/azure-pipelines/templates/core-build.yaml

```diff
@@ -36,6 +36,9 @@ steps:
     displayName: rush lint
     workingDirectory: ${{ parameters.workingDir }}
 
+  - script: node ./common/scripts/install-run-rush webpack:test -v
```

Comment:
Is this still needed? I see that you added the webpacking to the `build:ci` command.

Reply:
it's only needed for mac builds because mac doesnt run `build:ci`, i guess i could filter this to be run only on mac, but this only takes 20seconds, and the build should be cached for windows and linux

Reply:
gross... I won't block this PR for that but we should re-visit this. I'm still not a fan of build and build:ci